org.eclipse.lyo.oslc4j.core.plugin/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.lyo.tools</groupId>
+        <artifactId>tools-parent</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>org.eclipse.lyo.oslc4j.core.plugin</artifactId>
+    <packaging>pom</packaging>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.reficio</groupId>
+                <artifactId>p2-maven-plugin</artifactId>
+                <version>1.3.0</version>

```

Comment:
yes, that's the plugin I meant when you first discussed this with me - https://stackoverflow.com/questions/12476451/dependencies-from-pom-xml-not-considered-by-eclipse-in-tycho-project/24777560#24777560

but this approach creates only "dark" bundles with all the packages bundled together? not true OSGi modules with individual export declarations, right?

Reply:
Don't know what "dark" bundles mean. But it (hopefully) works.

The feature will contain a whole set of bundles for all the dependencies. As indicated on https://github.com/reficio/p2-maven-plugin, by default transitive dependencies will be fetched. And that's what I do here. But one can disable that too. Not sure if it is a good idea though.

Reply:
Exactly, that's what I mean by "dark" bundles. OSGi can resolve bundle deps at runtime like maven does at build time. Bundling them all together creates one large bundle where OSGi cannot control resolution. But yes, I know it works and is much easier than the other approach. I went for the reficio plugin when I realised how many deps I would have to package, in lyo it would be jena and co.

Reply:
But I can't figure out how to tell OSGI where to resolve these bundle dependencies from? What is the equivalent of maven-repo"?
Anyway, under "plugins" you can see what plugins are created https://download.eclipse.org/lyo/bundle/p2/superedge/.

What is missing is the JAX-RS implementation (jersey). Maybe this should be included as well, the same as I included explcitly slf4j.simple.